### DISCUSSION TITLE
Daily 06/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#543] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer no me sentía bien, así que no pude hacer nada.
Hoy empezaré a trabajar sobre las tareas que me asigné de back y de front.

**Comment by BraianTroncoso** [comment]
Ayer me puse a revisar con cual tarea iba a empezar, hoy las empiezo sin falta porque tuve un problema, quise migrar todo a Windows y tuve un conflicto con el disco solido, en fin, la cuestion es que no pude instalar python, opté por borrar algunos softwares en linux y ponerme a trabajar asi, borrando repositorios dependiendo en cual trabaje para hacerlo mas rapido y no perder tiempo. Ni bien terminemos el proyecto voy a optar por no tener mas dual e instalar linux definitivamente asi no reniego más.

**Comment by Brian-Almada** [comment]
Ayer me tomé el día porque llegaba mi esposa y mis niños del viaje y les dediqué el día. Recién hoy me voy a poner a ver que puedo hacer.

**Comment by rpgrca** [comment]
Hoy tuve que ir a reunión así que hasta la tarde no pude hacer nada. Estoy terminando lo de la selección, ya se está enviando al servidor el producto y se está agregando al carrito, y se graba en la base de datos. Quiero ver si puedo llegar a mostrar al carrito real antes de cortar hoy.
